tests/unit/browser/webengine/test_webenginetab.py

```diff
@@ -0,0 +1,109 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2018 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Test webenginetab."""
+
+from unittest import mock
+import logging
+
+from PyQt5.QtCore import QObject
+import pytest
+QtWebEngineWidgets = pytest.importorskip("PyQt5.QtWebEngineWidgets")
+QWebEnginePage = QtWebEngineWidgets.QWebEnginePage
+QWebEngineScriptCollection = QtWebEngineWidgets.QWebEngineScriptCollection
+
+from qutebrowser.browser.webengine.webenginetab import _WebEngineScripts
+from qutebrowser.browser.greasemonkey import GreasemonkeyScript
```

Comment:
I prefer only importing modules and not classes from modules (except for Qt, where it's clear because of the Q prefix) - that way it's clear where things are coming from.

In other words, this should be:

```python3
from qutebrowser.browser.webengine import webenginetab
from qutebrowser.browser import greasemonkey
```

and the usages should be adjusted accordingly.